### PR TITLE
Fixes: Enum of type byte is ignored #309

### LINKED ISSUE #309: Enum of type byte is ignored
**Problem**

Properties that use an enum type are ignored in the Swagger UI when that enum uses the **byte** type.

Consider this enum definition in C#:

```
  [JsonConverter(typeof(StringEnumConverter))]
  public enum ContactKind : byte
  {
    [EnumMember(Value = "phone")]
    Phone = 0,
    [EnumMember(Value = "mobile")]
    Mobile = 1,
    [EnumMember(Value = "fax")]
    Fax = 2,
    [EnumMember(Value = "email")]
    Email = 3,
    [EnumMember(Value = "web")]
    Web = 4
  }
```

It is ignored and not rendered in the Swagger UI.

When we change **byte** to **short**, it DOES work:

```
  [JsonConverter(typeof(StringEnumConverter))]
  public enum ContactKind : short
  {
    [EnumMember(Value = "phone")]
    Phone = 0,
    [EnumMember(Value = "mobile")]
    Mobile = 1,
    [EnumMember(Value = "fax")]
    Fax = 2,
    [EnumMember(Value = "email")]
    Email = 3,
    [EnumMember(Value = "web")]
    Web = 4
  }
```

**Expected behavior**

Render enums that use the byte type.

**Environment:**

 - Azure Functions App project with .NET 6 preview
 - Visual Studio 2022
 - The enum is defined in a .NET standard 2.1 class library
 - NuGet package: Microsoft.Azure.WebJobs.Extensions.OpenApi 0.9.0-preview

**Possible cause:**

In the OpenApi extension source code (https://github.com/Azure/azure-functions-openapi-extension/tree/main/src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors), I noticed these classes:
- Int16EnumTypeVisitor.cs
- Int32EnumTypeVisitor.cs
- Int64EnumTypeVisitor.cs

We probably need to add a **ByteEnumTypeVisitor.cs**?

Thank you!

### PR DESCRIPTION
**Pull request by emrekara37**
Resolves #309

### CODE REVIEW

File: src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/ByteEnumTypeVisitor.cs [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            var name = type.Key;
+
+            var instance = acceptor as OpenApiSchemaAcceptor;
+            if (instance.IsNullOrDefault())
+            {
+                return;
+            }
+
+            // Adds enum values to the schema.
+            var enums = type.Value.ToOpenApiIntegerCollection();
```
Comment (justinyoo) [inline src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/ByteEnumTypeVisitor.cs:52]: We need `ToOpenApiByteCollection()` method like `ToOpenApiInt16Collection()`

**Review by justinyoo — CHANGES_REQUESTED** [review]
@emrekara37 Thanks for the PR! I've left a comment on your change. Please take a look.

File: src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/ByteEnumTypeVisitor.cs (hunk 1 above)
Comment (emrekara37) [reply src/Microsoft.Azure.WebJobs.Extensions.OpenApi.Core/Visitors/ByteEnumTypeVisitor.cs]: Done.

**Comment by justinyoo** [comment]
@emrekara37 Thanks for the long wait! It's now merged.

**Pull request merged**